### PR TITLE
New example for Image-Segmentation

### PR DESCRIPTION
**Pull request by Senarect**
## Checklist

- [x ] I have read the **CONTRIBUTING** document
- [x ] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: examples/Image-Segmentation.ipynb [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Welcome to this tutorial! Have you done with the image segmentation  with crowdsourcing and now you need to somehow get the final results? Find out how to aggregate the data with methods from Crowd-Kit library by Toloka!"
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:20]: I've added a couple of comments for Text Aggregation example PR. Please fix ones that apply here too.

**Review by pilot7747 — CHANGES_REQUESTED** [review]

File: examples/Image-Segmentation.ipynb [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+        "Let's see the structure of our dataset"
+      ],
+      "metadata": {
+        "id": "JXak18NJovOy"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "df.columns"
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:127]: why not `df.head()`?

File: examples/Image-Segmentation.ipynb [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+        "This method performs a categorical aggregation task for each pixel: should it be included to the resulting aggregate or no. This task is solved by the single coin Dawid-Skene algorithm. Each worker has a latent parameter \"skill\" that shows the probability of this worker to answer correctly. Skills and true pixels' labels are optimized by the Expectation-Maximization algorithm."
+      ],
+      "metadata": {
+        "id": "ZEWzrvXRpQoo"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "result_EM = SegmentationEM().fit_predict(df)"
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:191]: `result_EM` -> `result_em`

File: examples/Image-Segmentation.ipynb [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "IOU_EM = np.array([])\n",
+        "for i in range(0, len(result_EM)-1):\n",
+        "  overlap = result_EM[i]*gt[i] \n",
+        "  union = result_EM[i] + gt[i] \n",
+        "  IOU_EM = np.append(IOU_EM, overlap.sum()/float(union.sum()))\n",
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:236]: `overlap.sum()/float(union.sum())` -> `overlap.sum() / float(union.sum())`

File: examples/Image-Segmentation.ipynb [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+        "id": "cznr36vlHYR4"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "IOU_MajorityVote = np.array([])\n",
+        "for i in range(0, len(result_MajorityVote)-1):\n",
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:296]: `len(result_MajorityVote)-1` -> `len(result_MajorityVote) - 1`

File: examples/Image-Segmentation.ipynb [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+        "  overlap = result_MajorityVote[i]*gt[i] \n",
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:297]: `result_MajorityVote[i]*gt[i]` -> `result_MajorityVote[i] * gt[i]`

File: examples/Image-Segmentation.ipynb [hunk 7]
```diff
@@ -0,0 +1,406 @@
...
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "IOU_EM = np.array([])\n",
+        "for i in range(0, len(result_EM)-1):\n",
+        "  overlap = result_EM[i]*gt[i] \n",
+        "  union = result_EM[i] + gt[i] \n",
+        "  IOU_EM = np.append(IOU_EM, overlap.sum()/float(union.sum()))\n",
+        "IOU_EM.mean()"
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:237]: `IOU_EM` -> `iou_em`

File: examples/Image-Segmentation.ipynb [hunk 8]
```diff
@@ -0,0 +1,406 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "IOU_MajorityVote = np.array([])\n",
+        "for i in range(0, len(result_MajorityVote)-1):\n",
+        "  overlap = result_MajorityVote[i]*gt[i] \n",
+        "  union = result_MajorityVote[i] + gt[i] \n",
+        "  IOU_MajorityVote = np.append(IOU_MajorityVote, overlap.sum()/float(union.sum()))\n",
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:299]: `IOU_MajorityVote` -> `iou_majority_vote`

Comment (pilot7747) [reply examples/Image-Segmentation.ipynb:299]: `overlap.sum()/float(union.sum())` -> `overlap.sum() / float(union.sum())`

File: examples/Image-Segmentation.ipynb [hunk 9]
```diff
@@ -0,0 +1,406 @@
...
+        "## Conslusion"
+      ],
+      "metadata": {
+        "id": "KCgyARgtMJu7"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "See? All the results are quite good but the SegmentationEM shows slightly better result.\n",
```
Comment (pilot7747) [inline examples/Image-Segmentation.ipynb:398]: `See?`. Please be less passive-aggressive

**Review by pilot7747 — CHANGES_REQUESTED** [review]

**Review by pilot7747 — APPROVED** [review]

File: examples/Image-Segmentation.ipynb [hunk 10]
```diff
@@ -0,0 +1,537 @@
...
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# How to Aggregate Image-Segmentation tasks via Crowdsourcing"
```
[the remark this answers is not in the document]
Comment (Senarect) [reply examples/Image-Segmentation.ipynb:20]: I do not undestand :c

[the remark this answers is not in the document]
Comment (Senarect) [reply examples/Image-Segmentation.ipynb:20]: No :c
There aren't any pictures in the dataset, only their masks

**Pull request merged**